implementations.md

```diff
@@ -0,0 +1,11 @@
+# Implementations of Container Specification
+
+## Runtime
+
+* [opencontainers/runc](https://github.com/opencontainers/runc) - reference implementation of OCI runtime
+* [hyperhq/runv](https://github.com/hyperhq/runv) - Hypervisor-based runtime for OCI
+
+## Testing
+
+* [huawei-openlab/ocp-testing](https://github.com/huawei-openlab/ocp-testing) - Testing framework for OCI configuration and runtime
```

Comment:
@vbatts  @wking @crosbymichael @mrunalp  Glad to have advise from you nice guys, any suggestion and considering are welcomed, expect more advices...
Thanks.